app/celery/tasks.py

```diff
@@ -266,6 +268,8 @@ def save_email(self, service_id, notification_id, encrypted_notification, sender
 
     if sender_id:
         reply_to_text = dao_get_reply_to_by_id(service_id, sender_id).email_address
+        if isinstance(template, tuple):
+            template = template[0]
```

Comment:
Do you see a way to add a test around it? I can handle it after we merge. Just want this code path to be covered.

Reply:
Can definitely be tested. You can mock redis and set up a service with a sender id